libvast/src/command.cpp

```diff
@@ -357,6 +357,34 @@ caf::error parse_impl(invocation& result, const command& cmd,
   return parse_impl(result, **i, position + 1, last, target);
 }
 
+void fixup_options(invocation& inv) {
+  using namespace std::string_literals;
+  auto move_option_arg = [&](auto cmd, auto format, auto opt) {
```

Comment:
Should this really take things by value?

Reply:
I think the deduced type should be `const char*`, so yeah.

Reply:
Care to make that explicit?